plugins/github/api.ml

```diff
@@ -341,6 +343,16 @@ module Commit_id = struct
         Some (Astring.String.concat ~sep:"/" branch)
       | _ -> None)
     | `PR _ -> None
+    
+  let pr_fork_branch_name t =
```

Comment:
What do you think about combining this functionality into the `branch_name` function above? Currently, it returns None for `PR` type.

Reply:
This would make sense to iff we can can always work with a PR fork head the same as way as we can with a branch in a `Ref`, otherwise we may not want to conflate them. I'm not sure off the top of my head whether they are interchangeable in all cases...

Reply:
Sorry for the delay in answering!

The branch names should be interchangeable, as long as we disambiguate a PR from a normal Ref and program accordingly.
It will break people's code where they expect this function to return `None` if the ref is a PR, though.

Reply:
Yes, I think so too.